src/terminal/adapter/InteractDispatch.cpp

```diff
@@ -142,7 +139,7 @@ bool InteractDispatch::MoveCursor(const VTInt row, const VTInt col)
     // Finally, attempt to set the adjusted cursor position back into the console.
     const auto api = gsl::not_null{ ServiceLocator::LocateGlobals().api };
     auto& info = ServiceLocator::LocateGlobals().getConsoleInformation().GetActiveOutputBuffer();
-    return SUCCEEDED(api->SetConsoleCursorPositionImpl(info, coordCursor));
+    FAILED_LOG(api->SetConsoleCursorPositionImpl(info, coordCursor));

```

Comment:
```suggestion
    LOG_IF_FAILED(api->SetConsoleCursorPositionImpl(info, coordCursor));
```

`FAILED_LOG` is the expression version of `LOG_IF_FAILED`